docs/howto/getting_started_markdown.md

```diff
@@ -0,0 +1,173 @@
+Getting Started Guide
+=====================
+
+Prerequisites
+-------------
+
+To get started with the Alteryx Core SDK, you need a valid installation
+of [Alteryx Designer](https://www.alteryx.com/).
+
+Python version 3.8.5 is required for Alteryx Designer plugins.
+
+Set Up the Development Environment
+----------------------------------
+
+Follow these steps to configure your development environment.
+
+### Create the Virtual Environment
+
+To get started\...
+
+1.  Install [Miniconda3](https://docs.conda.io/en/latest/miniconda.html)
+    for your system.
+2.  Once the install is done, open an Anaconda Prompt and create a new
+    virtual environment: `conda create -n ayx_python_sdk python=3.8.5`.
+3.  Next, activate it: `conda activate ayx_python_sdk`.
+
+If you are not familiar with Anaconda/Miniconda, visit [Anaconda
+Documentation](https://docs.anaconda.com/anaconda/user-guide/getting-started/).
+
+### Install the Package
+
+After you create and activate your virtual environment, pip install the
+Alteryx Core SDK pip package. For more information on pip, visit the
```

Comment:
TW review: same as above - Do we still call it Core SDK? Platform SDK? Python SDK? not sure which one we want to go with. @ayx-david-ung

Reply:
'Alteryx Platform SDK' is what we should be calling it